features/step_definitions/collection_steps.rb

```diff
@@ -212,21 +212,26 @@
   page.should have_content("Anonymous [#{authors}]")
 end
 
+# Scenarios using this step need the @javascript tag
```

Comment:
This is no longer true, right?

Reply:
correct, thanks! resolved in 186364e72d5a1338b46a584378e481f0b2142afb